### ISSUE TITLE
No ability to add NewerNoncurrentVersions with mc.

### ISSUE THREAD

**Issue by valterartur** [minio/mc#4137]
Hi!

I believe that I can't find anywhere in documentation regarding bucket lifecycle ability to set value for NewerNoncurrentVersions to my rule with minio client (mc). 

Here is example of output from my ilm rule:
```
mc --version
mc version RELEASE.2022-06-26T18-51-48Z (commit-id=40ee1a4ed60f3b2f618c4eef282ff8b29ace7045)
Runtime: go1.18.3 linux/amd64
Copyright (c) 2015-2022 MinIO, Inc.
License GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>


mc ilm edit --id test --noncurrentversion-expiration-days 1 --expired-object-delete-marker source/play
mc ilm ls source/play --json
{
 "status": "success",
 "target": "source/play",
 "config": {
  "Rules": [
   {
    "Expiration": {
     "DeleteMarker": true
    },
    "ID": "play",
    "NoncurrentVersionExpiration": {
     "NoncurrentDays": 1,
     "NewerNoncurrentVersions": 0
    },
    "Status": "Enabled"
   }
  ]
 }
}
```
So I can see that value for "NewerNoncurrentVersions" is defaulted to 0.
From help I can see that supported flags don't include any ability to add NewerNoncurrentVersions
```
SUPPORTED FLAGS:
   --id value                                          id of the rule to be modified
   --tags value                                        format '<key1>=<value1>&<key2>=<value2>&<key3>=<value3>', multiple values allowed for multiple key/value pairs
   --expiry-days value                                 the number of days to expiration
   --transition-days value                             the number of days to transition
   --storage-class mc-admin-tier-add                   storage class for current version to transition into. MinIO supports any warm tier configured via mc-admin-tier-add
   --expired-object-delete-marker                      remove delete markers with no parallel versions
   --noncurrentversion-expiration-days value           the number of days to remove noncurrent versions (default: 0)
   --noncurrentversion-transition-days value           the number of days to transition noncurrent versions (default: 0)
   --noncurrentversion-transition-storage-class value  storage class for noncurrent versions to transition into
   --config-dir value, -C value                        path to configuration folder (default: "/root/.mc")
   --quiet, -q                                         disable progress bar display
   --no-color                                          disable color theme
   --json                                              enable JSON lines formatted output
   --debug                                             enable debug output
   --insecure                                          disable SSL certificate verification
   --help, -h                                          show help
```

**Comment by harshavardhana** [comment]
@valterartur this feature is not added to CLI yet - it is currently available through APIs directly.

**Comment by krisis** [comment]
@valterartur until we add `NewerNoncurrentVersions` related flags to `mc ilm add` and `mc ilm edit`, you could construct an ILM policy json document containing rules with `NewerNoncurrentVersions` and apply them directly via `mc ilm import` like described [here](https://github.com/minio/minio/blob/master/docs/bucket/lifecycle/README.md#32a-automatic-removal-of-noncurrent-versions-keeping-only-most-recent-ones-immediately-minio-only-extension). Hope this helps.

**Comment by valterartur** [comment]
@krisis wow this is great WA! thanks, now I can drop s3cmd :)

**Comment by harshavardhana** [comment]
Fixed in #4153
